.gitignore

```diff
@@ -228,6 +228,7 @@ ClientBin/
 *.dbproj.schemaview
 *.jfm
 *.pfx
```

Comment:
can we remove this line instead of adding a new line?

Reply:
No. 
`pfx` the the certificate file type. By default, certificate should not be committed., just like secrets should not be committed.